### DISCUSSION TITLE
Using Public Facebook Videos

### DISCUSSION THREAD

**Discussion by pvperez1** [CollectionBuilder/collectionbuilder.github.io#51] [General]
Facebook is very popular here in the Philippines (whether I like it or not), so many organizations are uploading their videos on Facebook. I find that the code is very intuitive enough for me to modify it a bit to render a public facebook video, provided that the facebook video URL is correct (such as facebook.com/<user_id>/videos/<video_id> and must not be in the form of fb.com/<a_shortened_code> or facebook.com/watch/?v=<video_id>.

I'm pasting some codes and screenshots here:
- in video-embed.html: <img width="658" alt="image" src="https://user-images.githubusercontent.com/29356154/165892863-b2c95721-3f12-40e4-b170-04bfcac7f291.png">
- in video.html: 
<img width="703" alt="image" src="https://user-images.githubusercontent.com/29356154/165892948-d53816e6-5fc8-4d53-acdf-e36fb38ce63c.png">
- and the result: 
<img width="863" alt="image" src="https://user-images.githubusercontent.com/29356154/165892996-8a79f656-846e-40b8-9754-99f753921817.png">

**Comment by evanwill** [comment]
@pvperez1 thanks for the info! I think people will find this helpful. 

Eventually I would like to gather some of these sorts of customizations together in the docs some time, to show how you can add new item types in CB-CSV. 
(for future reference to remind myself: link to [facebook embedded videos docs](https://developers.facebook.com/docs/plugins/embedded-video-player))

**Reply by pvperez1** [reply]
Thank you @evanwill!
